### ISSUE TITLE
Warning "Tag 22.3-java17 was deleted or has expired" while building a native image

### ISSUE THREAD

**Issue by fedinskiy** [quarkusio/quarkus#30738]
### Describe the bug

Since the end of January 2023, during native build there is a warning "Tag 22.3-java17 was deleted or has expired". It looks like the default image `quay.io/quarkus/ubi-quarkus-mandrel:22.3-java17` was deleted.

### Expected behavior

No warning.

### Actual behavior

```
[WARNING] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Cannot find the `native-image` in the GRAALVM_HOME, JAVA_HOME and System PATH. Install it using `gu install native-image` Attempting to fall back to container build.
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildContainerRunner] Using docker to run the native image builder
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildContainerRunner] Checking image status quay.io/quarkus/ubi-quarkus-mandrel:22.3-java17
Error response from daemon: unknown: Tag 22.3-java17 was deleted or has expired. To pull, revive via time machine
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on native-image 22.3.0.1-Final Mandrel Distribution (Java Version 17.0.5+8)

```

### How to Reproduce?

1. `git clone git@github.com:quarkus-qe/quarkus-startstop.git -b 2.13`
2. `cd quarkus-startstop/app-jax-rs-minimal`
3. `mvn clean compile package -Pnative -Dquarkus.version=2.13.7.Final -Dquarkus.platform.group-id=io.quarkus.platform`

### Output of `uname -a` or `ver`

6.0.18-300.fc37.x86_64

### Output of `java -version`

17.0.5, vendor: GraalVM Community

### Quarkus version or git rev

2.13.7.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)

**Comment by gsmet** [comment]
@cescoffier ?

**Comment by zakkak** [comment]
Related to https://github.com/quarkusio/quarkus-images/issues/223

**Comment by cescoffier** [comment]
These are the old images. They are not really maintained. I will restore the tags, but we should also invite users to move to the new ones. I have no way to rebuild these images anymore.

**Comment by zakkak** [comment]
> but we should also invite users to move to the new ones.

AFAIK this is not possible for users of 2.13. Is that right?

**Comment by cescoffier** [comment]
I've restored the tag.

@zakkak yes, and the problem is that I'm stuck with the old image. I don't have a single machine able to build them. Tried two macs, 1 fedora, 1 ubuntu....  there is always something going wrong between docker, cekit, python, ruby... (my last hope is windows 10)

Same on github action.  Using containers didn't work either.

So, we may need to consider moving 2.13 to the new images.
